federated-learning/CLOUD.md

```diff
@@ -0,0 +1,59 @@
+# Federated Learning on Google Cloud Platform
+
+This guide will show you how to launch Google Cloud instances to run the federated learning tutorial in the cloud.
+
+If you want to run this part of the tutorial, you will need a [GCP account](https://cloud.google.com/) and to install [Google SDK](https://cloud.google.com/sdk/install).
+
+If not already done you'll need to clone the git repository `https://github.com/tf-encrypted/tf-encrypted`. These commands will be run from within the repo (`cd tf-encrypted/tools/gcp`).
+
```

Comment:
we should update the following comment since we have a copy of the cloud utils in `tf-world-tutorial`: 

```
If not already done you'll need to clone the git repository `https://github.com/tf-encrypted/tf-encrypted`. These commands will be run from within the repo (`cd tf-encrypted/tools/gcp`).
```

thanks

Reply:
done